library/Recurly/V3/API/Types/Coupon/DiscountType.hs

```diff
@@ -5,89 +5,86 @@ import Recurlude
 import qualified Data.Scientific as Scientific
 import qualified Numeric.Natural as Natural
 
-import qualified Recurly.V3.API.Types.Currency as Types
-import qualified Recurly.V3.API.Types.Money.Money as Types
-import qualified Recurly.V3.API.Types.Subscription.UnitAmount as Types
+import qualified Recurly.V3.API.Types.Currency as Currency
+import qualified Recurly.V3.API.Types.Money.Money as Money
+import qualified Recurly.V3.API.Types.Subscription.UnitAmount as Subscription
 
 -- | This should be a number between 1 and 100 inclusive. Zero is not allowed.
 -- Fractional parts of percents are not allowed. This number represents the
 -- percent discount to be applied at checkout. For example if you would pay $29
--- but you have a 10% discount (@CouponDiscountPercentValue 10@), you would
+-- but you have a 10% discount (@PercentValue 10@), you would
 -- only pay $26.10.
-newtype CouponDiscountPercentValue =
-  CouponDiscountPercentValue Natural.Natural
+newtype PercentValue = PercentValue Natural.Natural
```

Comment:
I'm not sure if it's pointless busywork or not, but: This one module defines many different types. Should the various types each move out to their own module? For example `...Coupon.DiscountType.PercentValue`.